### PR TITLE
Resolves Issue #7

### LINKED ISSUE #7: `-o` flag should also allow for path to location for file writing

### PR DESCRIPTION
**Pull request by gould-arthur**
Closes #7

### CODE REVIEW

File: pepmatch/matcher.py
Comment (danielmarrama) [inline pepmatch/matcher.py:859]: can we call `_get_function_pointer` `_get_output_function` instead?

File: pepmatch/tests/test_ouptput_files.py
Comment (danielmarrama) [inline pepmatch/tests/test_ouptput_files.py:58]: I think you'll need to actually create a full path like: `/full/path/to/file.csv`

**Review by danielmarrama — COMMENTED** [review]
Overally, very good attempt. Some minor changes and then I'll merge.

One other thing I'd like: 2 spaces instead of 4 for indentation 😃

**Review by danielmarrama — CHANGES_REQUESTED** [review]
Comment for requesting changes

File: pepmatch/matcher.py
Comment (gould-arthur) [reply pepmatch/matcher.py:859]: You bet

File: pepmatch/matcher.py
Comment (gould-arthur) [reply pepmatch/matcher.py:844]: I believe that df.to_tsv() does not exist, It is df.to_csv, with sep='\t'

Comment (danielmarrama) [reply pepmatch/matcher.py:844]: Yeah, I'm stupid. I've used this so many times and forgot that to_csv is used for .tsv so nvm this comment.

**Comment by gould-arthur** [comment]
Refactored the long lines near ln 800 in matcher.py
Only waiting for action approval, review on factored lines, and an answer to the question on absolute path.

File: pepmatch/matcher.py
Comment (danielmarrama) [inline pepmatch/matcher.py:810]: Just delete the first line and then I'll merge

**Review by danielmarrama — CHANGES_REQUESTED** [review]
One last change

**Comment by danielmarrama** [comment]
Very nice, Arthur. Thank you!!!!!!

**Pull request merged**
